### ISSUE TITLE
invalid metrics-datadog-0.1.3.pom

### ISSUE THREAD

**Issue by mkristian** [lookout/metrics-datadog#2]
https://bintray.com/artifact/download/lookout/systems/com/github/lookout/metrics-datadog/0.1.3/metrics-datadog-0.1.3.pom

is not a valid pom since the version with a `+` at the end do not exists:

``` xml
    <dependency>
      <groupId>com.indeed</groupId>
      <artifactId>java-dogstatsd-client</artifactId>
      <version>2.0.10+</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>org.apache.httpcomponents</groupId>
      <artifactId>fluent-hc</artifactId>
      <version>4.4+</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>com.fasterxml.jackson.core</groupId>
      <artifactId>jackson-databind</artifactId>
      <version>2.5.1+</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>org.mockito</groupId>
      <artifactId>mockito-all</artifactId>
      <version>1.9.5+</version>
      <scope>test</scope>
    </dependency>
```

and maven does not understand this syntax to be a version range.

**Comment by rtyler** [comment]
Silly gradle, I'll fix it

**Comment by rtyler** [comment]
published in 0.1.4
